tools/timemachine/internal/cmd/root.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This source code is provided 'as is' and no warranties are given as to title or non-infringement, merchantability
+// or fitness for purpose and, to the extent permitted by law, all liability for your use of the code is disclaimed.
+// This source code is governed by Apache License 2.0 that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use: "timemachine [command] [flags]",
+	Short: `timemachine is a command-line interface to go back specific block height.
+steps:
+1. mkdir ./tools/timemachine/etc
+   curl https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/v1.8.4/config_mainnet.yaml > ./tools/timemachine/etc/config.yaml
+   curl https://raw.githubusercontent.com/iotexproject/iotex-bootstrap/v1.8.4/genesis_mainnet.yaml > ./tools/timemachine/etc/genesis.yaml
```

Comment:
`2. refer to iotex-bootstrap to download config and genesis`
else we need to update it for every release

Reply:
fixed